### PR TITLE
PLANET-6592: Add minimal navigation option to Action page type

### PR DESCRIPTION
**Pull request by lithrel**
Ref: https://jira.greenpeace.org/browse/PLANET-6592
Goes with: https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/pull/800

Screenshot from 2022-02-23 18-18-07

## Test

Test with https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/pull/800 on [Sinope](https://www-dev.greenpeace.org/test-sinope/)
See https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/pull/800 for instructions

### CODE REVIEW

**Comment by planet-4** [comment]
### Test instance is ready :rocket:

:new_moon: [sinope](https://www-dev.greenpeace.org/test-sinope) | [admin](https://www-dev.greenpeace.org/test-sinope/wp-admin/) | [blocks report](https://www-dev.greenpeace.org/test-sinope/wp-admin/?page=plugin_blocks_report) | [CircleCI](https://app.circleci.com/pipelines/github/greenpeace/planet4-test-sinope) | [composer-local.json](https://github.com/greenpeace/planet4-test-sinope/blob/main/composer-local.json)

:watch: 2022.02.24 08:44:52

File: src/Context.php [hunk 1]
```diff
@@ -105,4 +105,37 @@ private static function get_campaign_scope( $global_project ) {
...
+	public static function set_custom_styles(
+		array &$context,
+		array $meta,
+		?string $post_type = null
+	): void {
+		if ( 'campaign' === $post_type ) {
+			$custom_styles = [
+				'nav_type'            => $meta['campaign_nav_type'] ?? null,
+				'nav_border'          => $meta['campaign_nav_border'] ?? null,
+				'campaign_logo_color' => 'green',
```
Comment (GP-Dan-Tovbein) [inline src/Context.php:123]: Just a doubt, but we're going to use the default green color? Or we should use the Greenpeace's green color?

Comment (lithrel) [reply src/Context.php:123]: This part is the legacy code for the Campaigns (a copy/paste of [this part](https://github.com/greenpeace/planet4-master-theme/pull/1633/files/c0f9f58d17d737a44cb8bb6a28e28f43e33d2854#diff-9aee92185c5b9b4bd23f4060ede93e8ea8b08c18b8b56643884c5ac8f31b118dL73-L86)), I think it should stay like that; for the Actions I assume the colors are controlled by the theme itself

**Review by GP-Dan-Tovbein — APPROVED** [review]
Code looks good to me! Tested and worked fine.

**Pull request merged**